data-service/src/main/resources/application-static.yaml

```diff
@@ -23,8 +23,7 @@ spring:
 spring:
   profiles: static-config
 
-# Discoveryless static configuration (presumably over localhost for integration-tests)
-# i.e. map from what-i-call-a-service -> service-URI
+# No Discovery static configuration
 web:
   client:
     audit-service: http://localhost:8081
```

Comment:
can we change audit service from 8081 to 8089/8098 for those on windows with port conflict issues on 81

Reply:
done